.github/workflows/create-release-branch.yml

```diff
@@ -41,9 +41,11 @@ jobs:
           # before Release Day.
           [[ $TODAY_DOW != 5 || $NEXT_RELEASE_DATE != $NEXT_TUESDAY_DATE ]] && echo "should_run=false" >> "$GITHUB_OUTPUT" || echo "should_run=true" >> "$GITHUB_OUTPUT"
   create-release-branch:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Switched to MacOS because the `base64` command on Linux behaves differently in ways I didn't expect.